Buildings/Resources/Data/Rooms/FFD/NaturalConvectionWithControl.dat

```diff
@@ -0,0 +1,321 @@
+0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```

Comment:
Why is this file needed? It looks like an output file?

Reply:
Those are what is called "obstacles files". They are needed to run the FFD code.

Reply:
OK then, but a bit surprising that so many zeros are needed to simply run code. Is this Windows specific?

Reply:
No. it is not Windows specific. There are bunch of such files in `Buildings/Resources/Data/Rooms/FFD. I could see them being retrieved in the C-code (`sci_reader.c`). The documentation says that they will be generated from a SCI program. the name of the original generated file is zeroone.dat so I guess they should have the zeros (maybe for the mesh definition?).